doc/manuals/sprokit/getting-started.rst

```diff
@@ -1,234 +1,485 @@
-In particular we'll need to know how to configure each process (the
-"Configuration") and how they can be hooked together (the input and
-output "Ports").
+	connect from filter.detected_object_set to writer.detected_object_set
 
-KWIVER comes with a sample
-[sprokit/pipelines/number_flow.pipe](sprokit/pipelines/number_flow.pipe)
-file that configures and connects the pipeline so that the ``numbers``
-process will generate a set of integers from 1 to 99 and the
-``print_number`` process will write those to a file called
-``numbers.txt``.  Of particular interest is the section at the end of
-the file that actually "hooks up" the pipeline.
+In this pipeline file we define three processes: reader, filter, and writer. We
+connect the detected_object_set output of reader to the detected_object_set
+input of filter. We configure filter to only pass detected_objects with a
+confidence above a threshold of 0.5 and then we pass its detected_object_set
+output port to our writer processes’ input port. We select a KPF writer for our
+writer process.
 
-To run the pipeline, we'll use the Sprokit ``pipeline_runner`` command::
+We can run the pipeline with the following command::
 
-    $ pipeline_runner -p </path/to/kwiver/source>/sprokit/pipelines/number_flow.pipe
+	pipeline_runner --pipe sample_reader_filter_writer.pipe
 
-After the pipeline completes, you should find a file, ``numbers.txt``, in your working directory.
+When the pipeline runs it will read a set of detected_objects from the file
+sample_detected_objects.csv, filter out any that have a confidence less than
+50%, and then write the remainder to a KPF file for further processing, etc.
 
 
-Python Processes
+PythonsProcesses
 ----------------
 
-One of KWIVER's great strengths (as provided by sprokit) is the
-ability to create hybrid pipelines which combine C++ and Python
-processes in the same pipeline.  This greatly facilitates prototyping
-complex processing pipelines.  To test this out we'll still use the
-``numbers`` process, but we'll use a Python version of the
-``print_number`` process called ``kw_print_number_process`` the code for
-which can be seen in
+One of KWIVER's great strengths (as provided by sprokit) is the ability to
+create hybrid pipelines which combine C++ and Python processes in the same
+pipeline.  This greatly facilitates prototyping complex processing pipelines.
+To test this out we'll  use a simple process called ``numbers`` which simply
+generates numbers on a Sprokit port.  We'll also use a simple Python process
+that prints the number  called ``kw_print_number_process`` the code for which
+can be seen in
 [sprokit/processes/python/kw_print_number_process.py](sprokit/processes/python/kw_print_number_process.py).
+
 As usual, we can lean about this process with the following command::
 
-    $ plugin_explorer --process --type kw_print_number_process -d --config
+	plugin_explorer --process --type kw_print_number_process -d --config
```

Comment:
I think the command whould be
 `plugin_explorer --proc kw_print_number_process -d`
or 
 `plugin_explorer --proc print -d`
The second may get more processes since it is matching on 'print'. The --config flag is no longer supported.